iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/MergeTest.java

```diff
@@ -227,8 +227,20 @@ private static Network createNodeBreakerWithVoltageAngleLimit(String nid) {
     }
 
     private static Network createNodeBreakerWithVoltageAngleLimit(NetworkFactory networkFactory, String nid, String valId) {
```

Comment:
Since finally no test was added / changed in this class, could you revert it in its initial state?

Reply:
It is fixed, thank you.